### PR TITLE
Add industry strength rl module

### PR DESCRIPTION
**Pull request by zhimin-z**
also adjust the space of table of modules for uniformity

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,14 +14,14 @@ This repository contains a curated list of awesome open source libraries that wi
 
 | | | |
 |-|-|-|
-|[🔍 Explaining Predictions & Models](#explaining-black-box-models-and-datasets) |[🔏 Privacy Preserving ML](#privacy-preserving-ml) | [📜 Model & Data Versioning](#model-and-data-versioning)|
-|[🏁 Model Training Orchestration](#model-training-orchestration)|[💪 Model Serving & Monitoring](#model-serving-and-monitoring)|[🤖 Neural Architecture Search](#neural-architecture-search)|
-| [📓 Data Science Notebook](#data-science-notebook) | [📊 Industry-strength Visualisation](#industrial-strength-visualisation) | [🔠 Industry-strength NLP](#industrial-strength-nlp) |
+| [🔍 Explaining Predictions & Models](#explaining-black-box-models-and-datasets) |[🔏 Privacy Preserving ML](#privacy-preserving-ml) | [📜 Model & Data Versioning](#model-and-data-versioning)|
+| [🏁 Model Training Orchestration](#model-training-orchestration)|[💪 Model Serving & Monitoring](#model-serving-and-monitoring)|[🤖 Neural Architecture Search](#neural-architecture-search)|
+| [🍕 Industry-strength RL](#industry-strength-rl) | [📊 Industry-strength Visualisation](#industrial-strength-visualisation) | [🔠 Industry-strength NLP](#industrial-strength-nlp) |
```
Comment (axsaucedo) [inline README.md:19]: Let's rename to "Reinforcement Learning"

**Review by axsaucedo — CHANGES_REQUESTED** [review]
This section looks very interesting, previously there has been an "unwritten rule" that in order to add a section there should be at least 3-5 projects, as otherwise we can end up with too much granularity, I would be keen to add this RL section, but I would be keen to make sure we add a couple of more resources. One that comes to mind is the Ray RL library.

Another point I would have is to call the section "Reinforcement Learning" instead of "industrial-RL" - I believe that NLP is more commonly referred to through its acronym, and although RL is a common acronym it may be better explicitly. Let's find a few more to add and then we can merge.

**Comment by axsaucedo** [comment]
I have updated CONTRIBUTING.md to reflect this https://github.com/EthicalML/awesome-production-machine-learning/blob/master/CONTRIBUTING.md

**Comment by zhimin-z** [comment]
> I have updated CONTRIBUTING.md to reflect this https://github.com/EthicalML/awesome-production-machine-learning/blob/master/CONTRIBUTING.md

Now take a look :) It meets all!

**Comment by zhimin-z** [comment]
> This section looks very interesting, previously there has been an "unwritten rule" that in order to add a section there should be at least 3-5 projects, as otherwise we can end up with too much granularity, I would be keen to add this RL section, but I would be keen to make sure we add a couple of more resources. One that comes to mind is the Ray RL library.
> 
> Another point I would have is to call the section "Reinforcement Learning" instead of "industrial-RL" - I believe that NLP is more commonly referred to through its acronym, and although RL is a common acronym it may be better explicitly. Let's find a few more to add and then we can merge.

I tweak a bit on the module title but keep it short for referential convenience. Is that fine?
image

**Comment by axsaucedo** [comment]
Yes I think most people would konw what RL is

**Pull request merged**